cmd/kubeadm/app/cmd/cmd.go

```diff
@@ -93,5 +95,7 @@ func NewKubeadmCommand(f cmdutil.Factory, in io.Reader, out, err io.Writer) *cob
 	experimentalCmd.AddCommand(phases.NewCmdPhase(out))
 	cmds.AddCommand(experimentalCmd)
 
+	utilfeature.DefaultFeatureGate.AddFlag(cmds.PersistentFlags())
```

Comment:
@luxas In terms of #49015, does it make sense to group features by cmd in `pkg/features`, or let the cmd group it in its own package? You mentioned something like `features.DefaultFeatures["kube-apiserver"]`, but I'm wondering what the pro of that is rather than delegating to the cmds.